### PR TITLE
avoid empty dict default value eval at function define time

### PR DESCRIPTION
**Pull request by swilly22**
#107

### CODE REVIEW

File: redisgraph/edge.py [hunk 1]
```diff
@@ -14,9 +14,9 @@ def __init__(self, src_node, relation, dest_node, edge_id=None, properties=None)
 
         self.id = edge_id
         self.relation = '' or relation
-        self.properties = {} or properties
         self.src_node = src_node
         self.dest_node = dest_node
+        self.properties = properties if properties is not None else {}
```
Comment (boris-42) [inline redisgraph/edge.py:19]: the true fix is actually just to switch places ```properties or {}``` would fix the thing

File: redisgraph/node.py [hunk 2]
```diff
@@ -4,14 +4,14 @@ class Node(object):
     """
     A node within the garph.
     """
-    def __init__(self, node_id=None, alias=None, label=None, properties={}):
+    def __init__(self, node_id=None, alias=None, label=None, properties=None):
```
Comment (boris-42) [inline redisgraph/node.py:7]: Yes this is very important to fix, as the are big side effects of this code.

File: redisgraph/edge.py [hunk 3]
```diff
@@ -14,9 +14,9 @@ def __init__(self, src_node, relation, dest_node, edge_id=None, properties=None)
 
         self.id = edge_id
         self.relation = '' or relation
```
Comment (gkorland) [inline redisgraph/edge.py:16]: @swilly22 don't you need for relation too?

Comment (boris-42) [reply redisgraph/edge.py:16]: https://github.com/RedisGraph/redisgraph-py/blob/master/redisgraph/edge.py#L19 it actually already fixed

**Pull request closed without merging**